learning-team/2020-04/2020-04-30.md

```diff
@@ -19,20 +17,45 @@ Add yourself to the list if you attend and check the box!
 
 ### Guests
 
-## Weekly Agenda
+Add yourself to the list if you attend and check the box:
+
+- [ ]
 
-- [ ]  Review action items (update weekly)
-- [ ]  Review [Initiatives](https://github.com/orgs/ember-learn/projects/19) [Board](https://github.com/orgs/ember-learn/projects/33) [](https://github.com/orgs/ember-learn/projects/19)(every second week)
-- [ ]  How is our PR queue doing? How many older than a week? [Tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)
-- [ ]  Review new RFCs/[RFC Tracking](https://github.com/emberjs/rfc-tracking)
-- [ ]  Any suggestions for the Ember Times? Add writeup ideas / call for contributors / suggestions to [the latest PR](https://github.com/ember-learn/ember-blog/pulls?q=is%3Aopen+is%3Apr+label%3A%22%F0%9F%97%9E+embertimes%22%20or%20#support-ember-times) or [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on Discord.
+## Topics - All-hands
 
-## TODOS
+### Review action items (weekly)
+See: https://github.com/orgs/ember-learn/projects/47.
 
+### Recurring - Review Initiatives (every second week)
+See: https://github.com/orgs/ember-learn/projects/19
 
-## Topics
+### Review Issue tracking sheet (weekly)
+See: https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454
 
-### Topic goes here (initials)
+### ADD YOUR TOPIC HERE (initials)
 
 ## Any Questions?
 Questions, comments or concerns? Submit a comment or PR for this set of notes.
+
+### Any suggestions for the Ember Times? (weekly)
+Any suggestions for the Ember Times? Add writeup ideas / call for contributors / suggestions to [the latest PR](https://github.com/ember-learn/ember-blog/pulls?q=is%3Aopen+is%3Apr+label%3A%22%F0%9F%97%9E+embertimes%22%20or%20#support-ember-times) or [#support-ember-times](https://discordapp.com/channels/480462759797063690/485450546887786506) on Discord.
+
+## Topics - Breakout - Infrastructure/Tooling
+
+### 404 errors on API docs
+
+## Topics - Breakout - Guides/API docs/Tutorials
+
+### Define criteria for which content should be published to Guides and which should be published to CLI Guides
+
+## Topics - Breakout - Website/Outreach
+
+### Redesign status
+
+### Ember Realworld Octane (CM 10mins)
+
+- https://github.com/gothinkster/ember-realworld/pull/99
+- https://github.com/gothinkster/ember-realworld/issues/100
+- Ember Octane Realworld was accepted: https://discordapp.com/channels/480462759797063690/480462759797063692/695383695976300606
+- Maybe it should replace the old version? https://discordapp.com/channels/480462759797063690/480462759797063692/695646337089405078
+
```

Comment:
Suggestion for the outreach breakout, see what would make sense to add to the Learn to Code collection:

```suggestion
### GitHub Learn to Code Collection
- https://github.com/collections/learn-to-code
```